frontend/libs/studio-components/src/components/StudioLabelWrapper/StudioLabelWrapper.module.css

```diff
@@ -0,0 +1,5 @@
+.studioLabelWrapper.withAsterisk::after {
+  content: '*';
```

Comment:
DS recommends to use `Må fylles ut` instead of `*` now, so we should change this